### PR TITLE
Adding J:bullettrain_side:

### PR DESCRIPTION
**Pull request by MATsxm**
Adding JTraining

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ shop | [shop.joomla.org](https://shop.joomla.org) | [report jshop](https://githu
 showcase | [community.joomla.org/showcase](http://showcase.joomla.org/) | [report jshow](https://github.com/joomla/joomla-websites/issues/new?title=[jshow]%20&body=Please%20describe%20the%20problem%20or%20your%20issue)
 sponsor | [joomla.org/sponsor.html](https://www.joomla.org/sponsor.html) | [report jspons](https://github.com/joomla/joomla-websites/issues/new?title=[jspons]%20&body=Please%20describe%20the%20problem%20or%20your%20issue)
 trademark | [tm.joomla.org](https://tm.joomla.org) | [report jtm](https://github.com/joomla/joomla-websites/issues/new?title=[jtm]%20&body=Please%20describe%20the%20problem%20or%20your%20issue)
+training | [joomla.org/joomla-training.html](https://joomla.org/joomla-training.html) | [report jtrain](https://github.com/joomla/joomla-websites/issues/new?title=[jtrain]%20&body=Please%20describe%20the%20problem%20or%20your%20issue)
```
Comment (mbabker) [inline README.md]: Isn't this on `community.joomla.org`?

Comment (jeckodevelopment) [inline README.md]: I guess it should be on training.joomla.org

**Comment by jeckodevelopment** [comment]
BTW, this URL `https://joomla.org/joomla-training.html ` gives a 404

File: README.md (hunk 1 above)
Comment (MATsxm) [inline README.md]: Right @mbabker 
My bad... need another coffee

**Pull request merged**